src/nwb_conversion_tools/datainterfaces/ophys/basesegmentationextractorinterface.py

```diff
@@ -38,13 +38,16 @@ def get_metadata_schema(self):
 
         # Temporary fixes until centralized definition of metadata schemas
         metadata_schema["properties"]["Ophys"]["properties"]["ImagingPlane"].update(type="array")
+        metadata_schema["properties"]["Ophys"]["properties"]["TwoPhotonSeries"].update(type="array")
```

Comment:
Yep, this is what I would have recommended from your original issue.

the `get_schema_from_hdmf_class` can provide a nice base filler of common items but so often we need to hand code little things like that in the `get_metadata_schema` methods.

Yet another reason in the end it might just be best to have all the metadata housed across hard-coded `.yml` files so it's all transparent, but also easy for the interface classes to read and use to set what they need.